services/webhook/notifier.go

```diff
@@ -440,20 +446,23 @@ func (m *webhookNotifier) DeleteComment(ctx context.Context, doer *user_model.Us
 	}
 
 	var eventType webhook_module.HookEventType
+	var pullRequest *api.PullRequest
 	if comment.Issue.IsPull {
 		eventType = webhook_module.HookEventPullRequestComment
+		pullRequest = convert.ToAPIPullRequest(ctx, comment.Issue.PullRequest, nil)
```

Comment:
```suggestion
		pullRequest = convert.ToAPIPullRequest(ctx, comment.Issue.PullRequest, doer)
```

Reply:
I have noticed numerous instances where the doer parameter needs to be added. Should I make these changes all at once?

Reply:
Done.